webpack/components/extensions/RegistrationCommands/fields/ActivationKeys.js

```diff
@@ -38,10 +38,16 @@ const ActivationKeys = ({
   };
 
   // Validate field when hostgroup is changed (host group may have some keys)
-  useEffect(() => {
+  useEffect(() => { 
```

Comment:
NIT: Trailing space, check the configuration of VS code, `Trim Trailing Whitespace` should be enabled by default if I am not mistaken